docs/logs/search-through-the-elasticsearch-api.md

```diff
@@ -9,6 +9,20 @@ When you use the API, here are the things you need to know:
   - host name: **logsene-receiver.sematext.com** / **logsene-receiver-syslog.eu.sematext.com** (if using Sematext Cloud Europe)
   - port: **80** (**443** for HTTPS)
   - index name: your [Logsene app token](https://apps.sematext.com/ui/logs) - note that this token should be kept secret
+  - apiKey provided in one of ways:
+    * basic auth credentials
+      ```
+      username:apiKey
+      password:31d28ff8-ae02-4ff9-b504-ea8013661412
+      ```
+  
+    * apiKey in `Authorization` HTTP header:
+      ```
+      Authorization: apiKey 31d28ff8-ae02-4ff9-b504-ea8013661412
+      ```
+
+
+    Note that `31d28ff8-ae02-4ff9-b504-ea8013661412` is just an example. You can find and regenerate you apiKey here: https://apps.sematext.com/ui/account/api)
```

Comment:
Instead of "here: <URL>" we can hyperlink URLs and add (US) or (EU) behind each of the two.